src/styles/components/_progress.scss

```
@@ -0,0 +1,47 @@
+$LINEAR_ANIMATION_TIME: 2.5s;
+
+@keyframes hold-visibility {
+  from { visibility: visible; }
+  to   { visibility: visible; }
+}
+
+@keyframes loading-progress-before {
+  from { transform: translate(-50%); }
+  to   { transform: translate(100vw) translate(-50%); }
+}
+
+.w-loading-progress {
+  background: #3fc4ff;
```

```suggestion
  background: $WEB_SECONDARY_COLOR;
```

You'll need to import the _colors.scss for this.

---

Done